index.js

```diff
@@ -0,0 +1,212 @@
+    </div>
+    </div>`;
+    existingCard0.appendChild(recentcard);
+  } else if (index < 4) {
+    const card = document.createElement('div');
+    card.classList.add('cards1');
+    card.innerHTML = `<div class="card-2 all-cards desk-card-${index}">
+  <div class="no-hover">
+  <h2 class="prof-art">${project.Title2} </h2>
+  <h2 class="desk-prof-art-dash">${project.Titled}</h2>
+  <p class="p22 desk-p22-d">${project.description}</p>
+  <ul class="skils desk-skils-d">
+  <li class="button desk-skil button-d">${project.technologies[0]}</li>
+  <li class="skil button desk-skil button-d">${project.technologies[1]}</li>
+  <li class="skil button desk-skil button-d">${project.technologies[2]}</li>
+  </ul>
+  </div>
+  <button class ="see-project-2 desk-see-project-1c" id="seeproject-2" onclick="popupp()">see project</button>
+  </div>
+  `;
+    existingCard.appendChild(card);
+  } else {
+    const card = document.createElement('div');
+    card.classList.add('cards1');
+    card.innerHTML = `<div class="card-2 all-cards desk-card-${index}">
+    <div class="no-hover">
+    <h2 class="prof-art">${project.Title2} </h2>
+    <h2 class="desk-prof-art-dash">${project.Titled}</h2>
+    <p class="p22 desk-p22-d">${project.description}</p>
+    <ul class="skils desk-skils-d">
+    <li class="button desk-skil button-d">${project.technologies[0]}</li>
+    <li class="skil button desk-skil button-d">${project.technologies[1]}l</i>
+    <li class="skil button desk-skil button-d">${project.technologies[2]}</li>
+    </ul>
+    </div>
+    <button class ="see-project-2 desk-see-project-1c" id="seeproject-2" onclick="popupp()">see project</button>
+    </div>
+    `;
+    existingCard2.appendChild(card);
+  }
+});
+
+let i = 0;
+ProjectsMobile.forEach((projec) => {
+  const popup = document.createElement('div');
+  popup.classList.add('modal');
+  popup.innerHTML = ` <div class="modal-content modal-contentD">
+      <div class="modalHead-close modalHead-closeD">
+      <h2 class = "modalHeader modalHeaderD" id="modalHeader">${projec.Title}${[i]}</h2>
+      <span class="close closeD">${projec.ClosePop}</span>
+      </div>
+      <div class = "desk0">
+      <ul id="modalTech" class = "modalTech">
+      <li>${projec.TecnologiesPop[0]}</li>
+      <li>${projec.TecnologiesPop[1]}</li>
+      <li>${projec.TecnologiesPop[2]}</li>
+      </ul>
+      <div class ="desk1">
+      <img class ="modal-img" src="${projec.ImagePop[0]}" alt="${projec.ImagePop[1]}">
+      <div class ="desk2">
+      <p class ="modal-pm">${projec.DescriptionPop} </p>
+      <div class = "live-source live-sourceD">
+      <a href="${projec.LinksPop[0][0]}">${projec.LinksPop[0][1]}<img src="${projec.LinksPop[0][2]}" alt="${projec.LinksPop[0][3]}"></a>
+  <a href="${projec.LinksPop[1][0]}">${projec.LinksPop[1][1]} <img src="${projec.LinksPop[1][2]}" alt="${projec.LinksPop[1][1]}"></a>
```

Comment:
- [ ] According to your previouse reviewer kindly Kindly adjust the spacing between the ` see live button` and `see source button` button to match the design on Figma, kidly fix this before you  resubmit for new review 🤗🛠

![Screenshot 2023-05-27 210254](https://github.com/solog0039/Portfolio-final/assets/92458236/ebc5fbd2-9c64-42e6-b4ca-9b3e8a2a3d61)